contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+- **Paths:** Which pages are they going to?
+- **Channels, referrers, and UTM parameters:** Where did they come from?
+- **Demographic information:** What devices, browsers, and operating systems are they using? Where are they located?
+- **Goals:** Are users performing the actions you want them to?
+
+We'll dive into all these below.
+
+## How do I use web analytics?
+
+### Visitors, pageviews, and sessions
+
+The first thing you'll see at the top of the web analytics dashboard is a graph of visitor data, which we can use to answer questions like these:
+
+- How many people are currently online _right now_?
+- How many people visited in the last week, month, or custom time period?
+- Is the number of visitors going up or down over a given time period?
+
+> **💡 Concept:** What is the distinction between a visitor and a session?
+> - A [**session**](/docs/data/sessions) is a set of events for a single visit to your website. Sessions have a beginning, a duration, and an end.
+> - A **visitor** represents a single person visiting your site.
+>
+> In other words, a single visitor will have many sessions as they return to your website. And each session will likely have many pageviews as they move from page to page.
+
+The first important feature to get familiar with is working with time intervals. You can adjust all the data on your web analytics dashboard to be scoped to the last 24 hours, the last 7 days, the year-to-date, or any other custom time range. In the screenshot below, we can see the last 30 days of traffic. You can also hover over the graph to see the value for the current period compared with the previous period.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/time_intervals_wide_2ce241b1c4.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/time_intervals_wide_dark_2256d417c6.png"
+  alt="Time interval selection"
+  classes="rounded"
+/>
+
+#### 📋 Use case: Finding when users are most active
+
+If you have a weekly newsletter, you may want to send it to your readers at a time they're likely to be active and not miss it. PostHog has a newsletter called [_Product for Engineers_](https://newsletter.posthog.com), and we can use these graphs to make an educated guess about when users are most active.
+
+- Filtering by the "Last 30 days" and grouping by day reveals that Tuesdays and Wednesdays are the most active weekdays for our users, whereas Saturdays are the days with the least activity.
+- Filtering by the "Last 7 days" and grouping by hour shows the ideal time to send is between 10:00 a.m. and 4:00 p.m. EST.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/last_7_days_light_8ca2fe6345.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/last_7_days_dark_d9273fa42c.png"
+  alt="Last 7 days by hour"
+  classes="rounded"
+/>
```

Comment:
Could we try detail expands for these again? I think this might work better than your original boxout approach, though they're fine as they are now.